modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: PROCEDURE
```

Comment:
These are supposed to be `:_content-type:`, not `:_module-type:` if you could update both files